db/seeds.rb

```diff
@@ -63,6 +64,71 @@
 description: "Deep inside an age-old mountain, clamber into a rustic mine train and make your way up a steep incline." 
 )
 
+### Review ###
+review_1 = Review.find_or_create_by(
+  title: 'Fun kids’ ride',
+  body: 'Emerge from the ocean’s depths in time to celebrate Ariel’s happily ever after in a spectacular fairytale finale.',
+  rating: 5,
+  user_id: 1,
+  ride_id: 1 
+)
+
+review_2 = Review.find_or_create_by(
+  title: 'Better than the movie',
+  body: 'Watch for angry hippos, hungry lions and “sleeping” zebras along the Nile and be on the lookout for a missing Jungle Cruise vessel and its helpless passengers. As the cruise continues down the Mekong River, you just might learn that the jungle always gets the last laugh.',
+  rating: 4,
+  user_id: 1,
+  ride_id: 2 
+)
+
+review_3 = Review.find_or_create_by(
+  title: 'It’s a 10-minute, 10,000-mile journey that you won’t soon forget!',
+  body: 'Steam past lush foliage, butterflies and waterfalls on the Amazon in South America. Glimpse an abandoned camp overrun by curious gorillas on the shores of the African Congo.',
+  rating: 4,
+  user_id: 2,
+  ride_id: 2 
+)
+
+### User ###
+# user_1 = User.find_or_create_by(
+#   email: "user1@gmail.com",
+#   username: "user1",
+#   password: "user1password"
+# )
+
+# user_2 = User.find_or_create_by(
+#   email: "user2@gmail.com",
+#   username: "user2",
+#   password: "user2password"
+# )
+
+# user_3 = User.find_or_create_by(
+#   email: "user3@gmail.com",
+#   username: "user3",
+#   password: "user3password"
+# )
+
+# user_4 = User.find_or_create_by(
+#   email: "user4@gmail.com",
+#   username: "user4",
+#   password: "user4password"
+# )
+
+# user_5 = User.find_or_create_by(
+#   email: "user5@gmail.com",
+#   username: "user5",
+#   password: "user5password"
+# )
+
+# user_6 = User.find_or_create_by(
+#   email: "user6@gmail.com",
+#   username: "user6",
+#   password: "user6password"
+# )
```

Comment:
we shouldn't leave commented out code in final submissions for someone to clean up later